main/test/flix/Test.Exp.Interpolation.flix

```diff
@@ -120,3 +215,60 @@ namespace Test/Exp/Interpolation {
         s.toUpperCase() as & Pure
 
 }
+
+namespace ToString {
+
+    pub class ToString[a] {
+        pub def toString(x: a): String
+    }
```

Comment:
Why isn't this causing an error for clashing with the existing ToString class?

Reply:
Ah-

We are slowly moving towards a situation where the compiler tests do not depend on the standard library. (This way it is much faster to run all tests). Most tests now are independent files that simply declare any constructs that they may need.

Reply:
Good question!